whitenoise/base.py

```diff
@@ -1,10 +1,7 @@
-from __future__ import absolute_import
```

Comment:
At first I incorrectly thought this was only required for Python 2.6, but it's required for 2.7 too. However, at least in Django itself we only include it where necessary so I added back the ones that were required here. I can add them all back if that's preferred, though I don't see an advantage to it.